test/labeling/model/test_class_balance.py

```diff
@@ -7,73 +7,80 @@
 from snorkel.model.utils import set_seed
 
 
+def generate_class_balance(k: int) -> np.ndarray:
+    """Generate class balance"""
+    p_Y = np.random.random(k)
+    p_Y /= p_Y.sum()
+    return p_Y
+
+
+def generate_cond_probs(
+    k: int, m: int, bias_diag: bool = True, abstains: bool = False
+) -> np.ndarray:
+    """Generate conditional probability tables for the m conditionally ind.
+    LFs, such that:
+        cpts[i, y1, y2] = P(\lambda_i = y1 | Y = y2)
+
+    Args:
+        k: (int) Number of classes
```

Comment:
nit: k = number of classes vs. cardinality throughout documentation

Reply:
Isn't the cardinality always just the number of classes though, so they're the same?